### ISSUE TITLE
[Adaptive Cards- Expenses Report]: Screen reader is conveying incorrect role as Link for the 'Export as PDF' button.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#9024]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: Dev (OS Build 27695.1000)
Browser Version: Version 130.0.2849.1 (Official build) dev (64-bit)
Screen Reader: Narrator

**Pre-Requisite:**
Turn on screen reader using caps + spacebar key.

**Repro steps:**
1. Open the above URL in the edge dev browser.
2. Navigate to the 'Sample' option and press enter key to activate it.
3. Press tab key and navigate to the 'Expenses Report' link and press enter key to activate it.
4. Press tab key and navigate to Export as PDF button and observe the issue.

**Actual Result:**
Screen Reader is conveying incorrect role information as Link for the 'Export as PDF' button. Currently role is defined as Link.

**Expected Result:**
Correct role should be defined as button so that when the focus lands on the 'Export as PDF' control then screen reader should announce the information as **'Export as PDF button.'**
 
**Note:**
Same issue is repro with NVDA as well.

**User Impact:**
It will impact the screen reader user as they will get confused if the incorrect role as link will be defined for any control as on activating it user will assume that new tab or page will be open.

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

https://github.com/user-attachments/assets/02eefd02-c0ef-408a-89e3-207409e47bff

![Incorrect role as Link is defined for the Export as pdf](https://github.com/user-attachments/assets/76268041-4e0f-44e2-b5f3-8782d3dd1733)

### Card JSON

```JSON
NA
```
